src/bp-core/admin/bp-core-admin-settings.php

```diff
@@ -1512,4 +1555,40 @@ function bp_admin_on_screen_notification_setting_tutorial() {
     </p>
 
 	<?php
-}
\ No newline at end of file
+}
+/**
+ * After update activity setting
+ *
+ * @since BuddyBoss 1.7.1
+ *
+ * @param string $tab_name  Settings tab name.
+ * @param object $class_obj Tab property.
+ *
+ * @uses bb_feed_post_types()                    Get all post type name.
+ * @uses bb_post_type_feed_option_name()         Settings option name for post type.
+ * @uses bb_post_type_feed_comment_option_name() Settings option name for post type comment.
+ *
+ * @return void
+ */
+function bb_after_update_activity_settings( $tab_name, $class_obj ) {
```

Comment:
```suggestion
function bb_after_update_activity_settings( $tab_name ) {
```
update doc block as well.